src/components/ProfileButtons/index.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
+
+import Icon from "components/shared/Icon";
+import getIcon from "util/getIcon";
+import { MovieButton, ButtonHolder } from "./styles";
+import API from "api/moviedb.instance";
+import { useAuth } from "contexts/AuthContext";
+import Alert from "components/shared/Alert";
+
+export default function ProfileButtons({ openTrailer, id, type }) {
+    const media = type === "movie" ? "Movies" : "TV";
+    const {
+        state: { user, session, userMedia },
```

Comment:
This is a bit complicated and bad practice to nest the deconstruction due to decrease in readability? How bout this:

```javascript
const { state: authState } = useAuth();
const { user, session, userMedia } = authState;
```